examples/c/udp-push-pop.c

```diff
@@ -58,7 +58,14 @@ static void server(int argc,
                    unsigned max_iterations)
 {
     int sockqd = -1;
+
+    #ifdef _WIN32
+    char expected_buf[DATA_SIZE];
+    #endif
+
+    #ifdef  __linux__
     char expected_buf[data_size];
+    #endif
```

Comment:
What is the rationale for making this different?

Reply:
The MSVC compiler does not allow using non-constant array length specifications, so I used the constant.

Reply:
Oh, I see. You mean that VLA is not supported by MSVC (even in c99 mode?) Anyways so Let's fix linux code so that it matches windows for this :)

Reply:
MSVC does not fully support C99 mode. It's mainly a C++ compiler which supports C as well. We can change the Linux code to match this, but that will take away the ability to change the `data_size` based on the program arg. Does that sound ok to you folks?